src/node/miner.h

```diff
@@ -19,6 +19,7 @@
 #include <boost/multi_index/tag.hpp>
 #include <boost/multi_index_container.hpp>
 
+class ChainstateManager;
```

Comment:
23517: we don't use `ChainstateManager` anywhere in `miner.*`. Bitcoin uses it in `RegenerateCommitments` which we don't have.

Reply:
Resolved in latest push!